engine/reference/commandline/scout_entitlement.md

```diff
@@ -0,0 +1,16 @@
+---
+datafolder: scout-cli
+datafile: docker_scout_entitlement
+title: docker scout entitlement
```

Comment:
Adding stubs is (currently) still a manual process, so if new commands are added, that needs to be done manually.